pkg/scheduler/framework/runtime/framework.go

```diff
@@ -538,7 +538,7 @@ func (f *frameworkImpl) RunFilterPlugins(
 			if !pluginStatus.IsUnschedulable() {
 				// Filter plugins are not supposed to return any status other than
 				// Success or Unschedulable.
-				errStatus := framework.NewStatus(framework.Error, fmt.Sprintf("running %q filter plugin for pod %q: %v", pl.Name(), pod.Name, pluginStatus.Message()))
+				errStatus := framework.AsStatus(fmt.Errorf("running %q filter plugin for pod %q: %w", pl.Name(), pod.Name, pluginStatus.AsError()))
```

Comment:
can you trace what happens with the error in higher function calls?

Maybe the name of the Pod is added again at some point.

Reply:
no, we use the function in both schedule and preempt and they are all clean.

Reply:
Get your point now. Nice catch!